new/mip_process_and_guidelines.md

```diff
@@ -0,0 +1,210 @@
+  * :arrow_right: Accepted (Core MIPs only) -- After the review end date, the Libertaria Developers will vote on whether to accept this change. If yes, the status will upgrade to Accepted.
+  * :arrow_right: Final (Not core MIPs) -- A successful Last Call without material changes or unaddressed complaints will become Final.
+* **Accepted (Core MIPs only)** -- This is being implemented by Libertaria Developers.
+  * :arrow_right: Final -- Standards Track Core MIPs must be implemented in at least two (2) viable Mercury clients before it can be considered Final. When the implementation is complete and supported by the community, the status will be changed to “Final”.
+* **Final** -- This MIP represents the current state-of-the-art. A Final MIP should only be updated to correct errata.
+
+Other exceptional statuses include:
+
+* Deferred -- This is for core MIPs that have been put off for a future hard fork.
+* Rejected -- An MIP that is fundamentally broken and will not be implemented.
+* Active -- This is similar to Final, but denotes an MIP which which may be updated without changing its MIP number.
+* Superseded -- An MIP which was previously final but is no longer considered state-of-the-art. Another MIP will be in Final status and reference the Superseded MIP.
+
+## What belongs in a successful MIP?
+
+Each MIP should have the following parts:
+
+- Preamble - RFC 822 style headers containing metadata about the MIP, including the MIP number, a short descriptive title (limited to a maximum of 44 characters), and the author details. See [below](https://github.com/libertaria-project/MIPs/blob/master/MIPS/mip-1.md#mip-header-preamble) for details.
+- Simple Summary - “If you can’t explain it simply, you don’t understand it well enough.” Provide a simplified and layman-accessible explanation of the MIP.
+- Abstract - a short (~200 word) description of the technical issue being addressed.
+- Motivation (*optional) - The motivation is critical for MIPs that want to change the Mercury protocol. It should clearly explain why the existing protocol specification is inadequate to address the problem that the MIP solves. MIP submissions without sufficient motivation may be rejected outright.
+- Specification - The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Mercury platforms, [and others](https://github.com/libertaria-project/wiki/wiki/Clients).
+- Rationale - The rationale fleshes out the specification by describing what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other languages. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussion.
```

Comment:
I think "how the feature is supported in other languages" is quite misleading here. We could maybe say "in design of other networks" because SIP, Hamachi, etc are somewhat similar despite their shortcomings.